creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -0,0 +1,125 @@
+const moment = require('moment')
+
+const redisClient = require('../redis')
+const { logger: genericLogger } = require('../logging')
+
+const SYNC_STATES = Object.freeze({
+  success: 'success',
+  fail: 'fail'
+})
+
+// Make key expire in 7 days in seconds
+const EXPIRATION = 7 /* days */ * 24 /* hr */ * 60 /* min */ * 60 /* s */
+
+/**
+ * Class intended to:
+ * - Record the number of the current day's successful and failed sync attempts
+ * - Record the timestamp of the most recent successful and failed sync
+ */
+
+// Note: When recording a sync 'success' or 'fail', it implies that the secondary was able to respectfully
+// successfully or unsuccessfully take the primary's exported data, and update its own state
+
+class SyncHistoryAggregator {
+  static async recordSyncSuccess (logContext) {
```

Comment:
some jsdoc or at least a one-line-comment on these functions would be helpful

Reply:
added

Reply:
thanks!